libs.osgi/external/binaries-list

```diff
@@ -14,5 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-972E6455724DC6ADB1C1912F53B5E3D7DF20C5FD osgi.cmpn-4.2.jar
-2F2FBFF3FB80F91B159A7FA9F0C42A894BB8BCC5 osgi.core-5.0.0.jar
+972E6455724DC6ADB1C1912F53B5E3D7DF20C5FD org.netbeans.external:osgi.cmpn:4.2
```

Comment:
How does this work? I don't see this on maven central. There are osgi.cmpn artifacts in the org.osgi group, just different versions. I would use this opportunity to move to the "primary" ones.

Reply:
I'd rather wait until later to do this. This co-ordinate works (http://mvnrepository.com/artifact/org.netbeans.external/osgi.cmpn-4.2/RELEASE82) and is closer to the original entry in binaries-list than the primary equivalent.

Reply:
Your link shows, that the referenced binary is not present in maven central, but only in the netbeans repository (see the "Repositories" row). The download binaries task has maven central hard-coded, so the binary won't be found.

It works for you (and me), because there are multiple fallbacks:
- local maven2 cache ($HOME/.m2/repository)
- fallback to load the file from `http://hg.netbeans.org/binaries/`

I would not change the binary reference, as currently there is no valuable alternative. And I can support your assessment not to change the binary right now. This makes it clear, that this needs to be tackled later.

Reply:
OK, thanks, merging.